sim.html

```diff
@@ -378,16 +392,38 @@
 			setInterval(updateClock, 60*1000);
 
 			// Online/offline toggle
-			document.querySelector('input#online').addEventListener('change', function(e) {
+			document.querySelector('#online').addEventListener('click', function(e) {
+				const input = document.getElementById('online').querySelector('input')
+				const checked = !input.checked
 				Object.defineProperty(appFrame.contentWindow.Navigator.prototype, "onLine", {
 			        enumerable: true,
 			        configurable: true,
 			        get: function() {
-			        	return e.target.checked;
+			        	return checked;
 			        }
 			    });
-			    appFrame.contentWindow.dispatchEvent(new Event(e.target.checked ? 'online' : 'offline'));
+			    appFrame.contentWindow.dispatchEvent(new Event(checked ? 'online' : 'offline'));
+				input.checked = checked
 			});
+
+			// Experiment Group for Trending Articles toggle
+			document.querySelector('#experiment-trending').addEventListener('mousedown', function(e) {
+				const input = document.getElementById('experiment-trending').querySelector('input')
+				const checked = !input.checked
+				if ( checked ) {
+					localStorage.setItem( '2021-KaiOS-app-engagement-config', `{"timestamp":${Date.now()},"startDate":"20210101","endDate":"20211231","countries":["NG"],"languages":["en"]}` )
+					localStorage.setItem( '2021-KaiOS-app-homepage-content-suggestions', 'trending-articles')
+					localStorage.setItem( 'user-country', 'NG' )
+				} else {
+					localStorage.removeItem( 'user-country' )
+				}
+				input.checked = checked
+			});
+
+			// initial value of the experiment toggle checkbox
+			if ( localStorage.getItem( 'user-country' ) ) {
```

Comment:
Not sure about this condition. When I load the sim this box is checked because I have a user-country setup but I don't meet all the other conditions for the experiment to be ON. It could be unchecked by default. It would not be perfect but it would be much better.